aea/manager/manager.py

```diff
@@ -415,6 +413,7 @@ def add_project(
                 f"Failed to load project: {public_id} Error: {str(e)}", e
             )
 
+        self._versionless_projects_set.add(public_id.to_any())
```

Comment:
This is what caused the pesky error @panasevychol . When the project failed to load, the second time we would not be able to add it due this being placed wrongly.